collectors/bzimport/constants.py

```diff
@@ -26,3 +26,5 @@
 
 # Maximum age of connection to Bugzilla in seconds, recommended value is 60
 BZ_MAX_CONNECTION_AGE = get_env("BZ_MAX_CONNECTION_AGE")
+
+COLLECTOR_ENABLED = get_env("BZ_COLLECTOR_ENABLED", default="True", is_bool=True)
```

Comment:
Could we make this on a higher level of granularity? Currently eg. on the UAT we would need the flaw collector disabled by the tracker and the metadata one enabled which is not possible with this simple approach.